e2e/steps_file.js

```diff
@@ -196,6 +197,7 @@ module.exports = function () {
 
     async addAnotherElementToCollection(collectionName) {
       const numberOfElements = await this.grabNumberOfVisibleElements('.collection-title');
+      this.waitForElement('.form-group .button');
```

Comment:
maybe this one should be line above?

Reply:
removed it, turned out that `addAnotherElementToCollection` was not working for scanned documents